setup.py

```diff
@@ -21,8 +21,8 @@
     "scipy",
     "numba",
     "thewalrus>=0.17.0",
-    "tensorflow>=2.4.0",
-    "tensorflow-probability",
+    "tensorflow<=2.10.1",
```

Comment:
I had to comment this out and `python -m pip install tensorflow-macos=2.10.0`

Reply:
I'm not sure how to resolve the issue on the `setup.py` file. Poetry would help with this.

Reply:
try importing `platform` (it's a built-in module) and then use the line `"tensorflow_macos==2.10.0" if platform.system() == "Darwin" else "tensorflow==2.10.1"`

Reply:
actually I can just try and see if it works